src/tests/ftest/container/full_pool_container_create.py

```diff
@@ -86,32 +88,56 @@ def test_no_space_cont_create(self):
             "Pool %s query data: %s\n", self.pool.uuid, self.pool.query_data)
 
         # create a 2nd container now that pool is full
-        try:
-            self.log.info("creating 2nd container")
-            cont2 = DaosContainer(self.context)
-            cont2.create(self.pool.pool.handle)
-            self.log.info("created 2nd container")
+        # try writing to this second container which should
+        # fail after writing a few Kb of data as there should
+        # be some space emptied up due to aggregation and
+        # rdb log compression
+        self.log.info("creating 2nd container")
+        cont2 = DaosContainer(self.context)
+        cont2.create(self.pool.pool.handle)
+        self.log.info("created 2nd container")
+        self.log.info("opening container 2")
+        cont2.open()
+
+        written_data_cont2 = 0
+        self.log.info("writing more objects, write expected to fail "
+                      "after writing a few Kb")
+        for obj_sz in [10, 1]:
+            write_count = 0
+            while True:
+                try:
+                    # write to second container
+                    cont2.write_an_obj(my_str, obj_sz, dkey, akey,
+                                       obj_cls="OC_SX")
+                    write_count += 1
 
-            self.log.info("opening container 2")
-            cont2.open()
+                except DaosApiError as excep:
+                    if not (err in repr(excep) or err2 in repr(excep)):
```

Comment:
So, in this case, it supposes to fail with 1007 and not 1009, If that is the case maybe we should remove err2?

Reply:
Thanks for pointing. I have verified we don't need to check for -1009 anymore and have removed it from the code.